com.unity.render-pipelines.universal/ShaderLibrary/SSAO.hlsl

```diff
@@ -319,63 +375,58 @@ float4 SSAO(Varyings input) : SV_Target
 // Geometry-aware separable bilateral filter
 half4 Blur(float2 uv, float2 delta) : SV_Target
 {
-    float4 p0 = SAMPLE_BASEMAP(uv                 );
-    float4 p1a = SAMPLE_BASEMAP(uv - delta * 1.3846153846);
-    float4 p1b = SAMPLE_BASEMAP(uv + delta * 1.3846153846);
-    float4 p2a = SAMPLE_BASEMAP(uv - delta * 3.2307692308);
-    float4 p2b = SAMPLE_BASEMAP(uv + delta * 3.2307692308);
+    half4 p0 =  (half4) SAMPLE_BASEMAP(uv                 );
+    half4 p1a = (half4) SAMPLE_BASEMAP(uv - delta * 1.3846153846);
```

Comment:
How are these constants obtained? Are these UV adjustments to use bilinear sampling to do part of the blur? We should have a named constants for them.

Reply:
Noted! I'll adress this in the Blur PR for SSAO.